docs/src/index.md

```diff
@@ -6,6 +6,8 @@ This resource aims to teach you everything you need
 to know to get up and running with tabular data manipulation using the DataFrames.jl package
 and the Julia language.
 
+This documentation is always updated. There is available also a [DataFrames Tutorial in NoteBooks](https://github.com/bkamins/Julia-DataFrames-Tutorial/). That tutorial is not guarantee that be completely updated, but it is a good complementary resource to learn about DataFrames usage. 
```

Comment:
Maybe don't insist on the risk of the tutorial being out of date, given that it's maintained by @bkamins. But mentioning that the tutorial illustrates other packages would make sense.

Reply:
Yes, you are right. I have removed the mention to the risk of being out of date, and add that "it illustrates its usage in conjunction with other packages.".

Reply:
Also, @bkamins  I was referring only to make more accesible the Tutorial, I understand the aim of avoid the additional dependences. Only I think that the Tutorial is a great learning material, and it has a lot of sense mention it in the Documentation.